client/src/test/diagnostics.test.ts

```diff
@@ -715,41 +641,27 @@ describe('Should get diagnostics for invalid extensions in ⚡4ads page', () =>
       {
         message:
           "⚡4ads : The extension 'amp-animation' was found on this page, but is unused. Please remove this extension.",
-        range: toRange(119, 20, 119, 25),
+      range: toRange(119, 25, 119, 20),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡4ads',
         code: 'EXTENSION_UNUSED',
       },
       {
         message:
           "⚡4ads : The extension 'amp-lightbox' was found on this page, but is unused. Please remove this extension.",
-        range: toRange(119, 20, 119, 25),
+      range: toRange(119, 25, 119, 20),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡4ads',
         code: 'EXTENSION_UNUSED',
       },
       {
         message:
           "⚡4ads : The extension 'amp-mustache' was found on this page, but is unused. Please remove this extension.",
-        range: toRange(119, 20, 119, 25),
+      range: toRange(119, 25, 119, 20),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡4ads',
         code: 'EXTENSION_UNUSED',
       },
-      {
-        message: 'Doctype must be declared first.',
-        range: toRange(0, 0, 0, 4),
-        severity: vscode.DiagnosticSeverity.Warning,
-        source: null,
-        code: 'doctype-first',
-      },
-      {
-        message: '<title> must be present in <head> tag.',
```

Comment:
Why is this removed?

Reply:
This error does not exist in the latest validator.